docs/customizations/playground-examples.md

```
@@ -0,0 +1,62 @@
+---
+title: Playground Examples API
+---
+
+<script setup>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: script

(MD033, no-inline-html)